intermine_boot/intermine_docker.py

```diff
@@ -222,16 +230,17 @@ def create_intermine_builder_container(client, image, env, options):
         'MINE_NAME': os.environ.get('MINE_NAME', 'biotestmine'),
```

Comment:
you have to change this to extracted folder name too for it to work with the above changes.

Reply:
done.